### PR TITLE
Cockpit 181

### PR DESCRIPTION
**Pull request by mvollmer**
Rendered: https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-181.html

### CODE REVIEW

File: _posts/2018-10-31-cockpit-181.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
+
+The appearance of memory and CPU graphs has been improved, and their
+colors are more accessible.
+
+![New memory graph](/images/memory-graph.png)
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: There is an awful amount of empty white space around that PNG, can this get trimmed?

File: _posts/2018-10-31-cockpit-181.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Memory on Linux is often hard to understand and usually presented
+incorrectly in most tools. There are so many different terms, most of
+which do not matter to anyone who's not a kernel developer.
+
+In addition, Cockpit's old memory graphs were unfortunately calculated
+incorrectly as well.
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: The "In addition" sounds wrong. How about dropping this sentence, and fold it into the above, like "usually presented incorrectly in most tools, including Cockpit's memory graphs".

File: _posts/2018-10-31-cockpit-181.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+### Graph improvements
+
+The appearance of memory and CPU graphs has been improved, and their
+colors are more accessible.
+
+![New memory graph](/images/memory-graph.png)
+
+Memory on Linux is often hard to understand and usually presented
+incorrectly in most tools. There are so many different terms, most of
+which do not matter to anyone who's not a kernel developer.
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: I'd leave out this sentence, it's not really descriptive/specific and might even be interpreted as derogatory.

File: _posts/2018-10-31-cockpit-181.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+slug: cockpit-181
+category: release
+summary: ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: We usually prefix this with the affected page. "System: Memory graph improvements"

File: _posts/2018-10-31-cockpit-181.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+Why do we not show cache on the overview and de-emphaize it on the
+detailed memory view? Linux stores often-used files in memory as cache
+to speed up performance. As cache is quickly reclaimed when needed,
+it's effectively available RAM. That's why Cockpit shows it a free
+space, as Linux considers it free too.
+
+Additional information on can be found on the website [Linux Ate My
+RAM](https://www.linuxatemyram.com/).
+
+###  Edit network interfaces of virtual machines
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: "Networking:" prefix

File: _posts/2018-10-31-cockpit-181.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+The configuration of the network interfaces of a virtual machine can
+now be changed in Cockpit.
+
+![nic dialog](/images/nic-edit.png)
+
+### Update look of lists to match Patternfly
+
+...
+
+### Followup fixes related to the switch away from react-lite
```
Comment (martinpitt) [inline _posts/2018-10-31-cockpit-181.md]: Traditionally we don't document bug fixes; we always have so many of them. I'd just leave this out.

File: _posts/2018-10-31-cockpit-181.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2018-10-31-cockpit-181.md]: @garrett already did this on his fork.

Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Yes, also the nic editing screenshot needs some cropping.

File: _posts/2018-10-31-cockpit-181.md (hunk 5 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: It's "Machines: ", I'd say.

File: _posts/2018-10-31-cockpit-181.md (hunk 2 above)
Comment (garrett) [reply _posts/2018-10-31-cockpit-181.md]: Yep, sounds good.

File: _posts/2018-10-31-cockpit-181.md (hunk 3 above)
Comment (garrett) [reply _posts/2018-10-31-cockpit-181.md]: Yeah, good point. :+1:

File: _posts/2018-10-31-cockpit-181.md (hunk 4 above)
Comment (garrett) [reply _posts/2018-10-31-cockpit-181.md]: :+1:

File: _posts/2018-10-31-cockpit-181.md (hunk 6 above)
Comment (garrett) [reply _posts/2018-10-31-cockpit-181.md]: Does switching to React from React-lite bring any user-facing updates? If not (and I think that's the case here), I agree.

File: _posts/2018-10-31-cockpit-181.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Done

File: _posts/2018-10-31-cockpit-181.md (hunk 4 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Done.

File: _posts/2018-10-31-cockpit-181.md (hunk 5 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Done

File: _posts/2018-10-31-cockpit-181.md (hunk 6 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Removed

File: _posts/2018-10-31-cockpit-181.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2018-10-31-cockpit-181.md]: Removed

**Comment by garrett** [comment]
It's a bit odd to mention CPU graph improvements under the memory graph area, but otherwise LGTM. 

Memory graphs were the focus, and CPU display enhancements "[rode in on the coattails]"(https://idioms.thefreedictionary.com/ride+coattails) (also benefited from the improvements), so it's OK.

**Pull request merged**